modules/openapi-generator/src/main/resources/C-libcurl/object-header.mustache

```diff
@@ -13,7 +13,7 @@
 
 
 typedef struct object_t {
-    void *temporary;
+    cJSON *json;
```

Comment:
Why `void *` needs to be changed to `cJSON *`? May I know what requirements object.h and object.c cannot meet?

Reply:
None at all, I explained that in the commit message. This patch was written at a time when object_t was broken, before you fixed it yourself. So it was easier for us to just keep working with our version, which we had already tested. We didn't think anyone would mind, since so many changes are needed anyway, but I can use your version of object_t if you prefer.

Reply:
It's good to keep the current code. Actually this is not my original. I took the code from https://github.com/OpenAPITools/openapi-generator/pull/12069/commits/166de171e78994ce0ba0783bd1a651b03310da0c and found it to work fine. So let's keep the code.

Reply:
If needed we could modify it as a union so we can accommodate int, char, void* data types as well